android/Getting-app-logs-from-Android-Studio.md

```diff
@@ -5,10 +5,4 @@ Brief walkthrough for users on how to post logs if they are encountering issues.
 1. On your computer, [download](https://developer.android.com/studio/) and [install](https://developer.android.com/studio/install) Android Studio
 2. On your phone, [enable developer options and USB debugging](https://developer.android.com/studio/debug/dev-options#enable)
 3. Connect your phone to your computer with a USB cable
-4. Open Android Studio, and click `Android Device Monitor`
-![android_device_monitor_1](https://user-images.githubusercontent.com/3611199/27092050-6bba06c2-50a5-11e7-9123-909d6fdc52ac.png)
-5. In Android Device Monitor, click on the name of your phone.
-6. In the text field in the middle of Android Device Monitor, set the drop down menu on the right to "debug"
-![android device monitor](https://user-images.githubusercontent.com/3611199/27092086-8dbc3c86-50a5-11e7-9b1e-91f1f675ec72.png)
-7. On your phone, open the Commons app and perform whatever action you are having problems with in the app
-9. Copy all of the logs from Android Device Monitor (in the box on the bottom right) and paste them into an issue report on [Github](https://github.com/commons-app/apps-android-commons/issues/new), along with a description of the problem you are having.
```

Comment:
I wonder if it would've been useful to still keep these lines.

Reply:
The last lines have been erased because the Android official docs have all the information illustrated elaborately. 
![image](https://user-images.githubusercontent.com/37215508/82861925-d7c32a00-9f3b-11ea-9be3-1a1f124e28e6.png)
However, since we are talking about this, I think we can keep the last point which says, "place the logs in issue report" for our contributors. @sivaraam Please enlighten on this and I will make the changes accordingly :)

Reply:
Yeah, I get that. I think the last two point still make sense to be included even though we link to the official docs. So, make a change that includes those points again. I believe it would read better. We can confirm that through a PR discussion.

Reply:
Okay, sure. I will make a new pr on it.